tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "skipLibCheck": true,
+    "lib": ["es2022"],
+    "target": "es2022",
+    "module": "Node16",
+    "moduleResolution": "node16",
+
+    "moduleDetection": "force",
+    "isolatedModules": true,
+    "verbatimModuleSyntax": true,
+
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+
+    "noEmit": true,
```

Comment:
I copied the file from spark-stats repo, but changed the configuration of what TypeScript emits. In other repos, `tsc -p .` writes `.d.ts` files to the `dist` folder. That's useful for libraries. We are building apps (services) that are not published to npm, therefore I don't see any value in creating `.d.ts` files.

https://github.com/filecoin-station/spark-stats/blob/8a3ec03142d5268b96d2846cf2d81efea7c158bc/tsconfig.json#L19-L22